phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/cmd/ResnikCommand.java

```diff
@@ -1,17 +1,18 @@
 package org.monarchinitiative.phenol.cli.cmd;
 
 
-import org.monarchinitiative.phenol.cli.demo.ResnikGenebasedHpoDemo;
+import org.monarchinitiative.phenol.cli.demo.ResnikGeneBasedHpoDemo;
 import picocli.CommandLine;
 
+import java.io.IOException;
 import java.nio.file.Path;
 import java.util.concurrent.Callable;
 @CommandLine.Command(name = "resnik",
   mixinStandardHelpOptions = true,
   description = "Resnik")
 public class ResnikCommand implements Callable<Integer> {
 
-  @CommandLine.Option(names = {"--hpo"}, description = "path to hp.obo file", required = true)
+  @CommandLine.Option(names = {"-h"}, description = "path to hp.obo file", required = true)
```

Comment:
just add `-h` - don't break people's code!

Also, should this now read `path to hp.json file` ?